.github/workflows/module-tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       fail-fast: false
       matrix:
         config: [
-          config-empty,
+          config-hdp2,
```

Comment:
hdp2 is not well descriptive, because we maintain HDP 2.6 and HDP 2.5 images

i understand this way it's consistent... up to you